Backstories/Backstories.xml

```diff
@@ -2338,7 +2338,7 @@ When HE was informed of his assignment to the Xennoa-Zartza War, HE decided HE d
 	<Explorer36>
 		<title>Explorateur/rice</title>
 		<titleShort>Explorateur</titleShort>
-		<desc>NAME voyagea de planète en planète, collectant des échantillons, des histoires et des connaissances. HE esperait un jour pouvoir compiler un registre de chaque animal, plante, minéral et machine et appareil de l'espace connu : Le Parfait Guide du Voyager dans les RimWorlds.</desc>
```

Comment:
J'ai traduit comme ça parce que c'est effectivement comme ça que c'est initialement : "mineral, and machine and device in known space". Je trouve aussi que c'est un peu bizarre mais ça fait sens et c'est pas faux, donc à voir.

Reply:
En français, selon moi, mettre deux "et" dans une énumération est une erreur. D'ailleurs, si tu lis la phrase à voix haute, ça cloche. Mais tu peux modifier si tu es sûr de toi.

Reply:
On est d'accord, on peut garder comme ça vu que le sens ne change pas de toute façon.